downloads/swan-lake-release-notes/swan-lake-beta4/RELEASE_NOTE.md

```diff
@@ -0,0 +1,831 @@
+---
+layout: ballerina-left-nav-release-notes
+title: Swan Lake Beta4
+permalink: /downloads/swan-lake-release-notes/swan-lake-beta4/
+active: swan-lake-beta4
+redirect_from: 
+    - /downloads/swan-lake-release-notes/swan-lake-beta4
+    - /downloads/swan-lake-release-notes/
+    - /downloads/swan-lake-release-notes
+---
+## Overview of Ballerina Swan Lake Beta4
+
+<em>This is the fourth Beta release in a series of planned Alpha and Beta releases leading up to the Ballerina Swan Lake GA release.</em> 
+
+It introduces the new language features planned for the Swan Lake GA release and includes improvements and bug fixes done to the compiler, runtime, standard library, and developer tooling after the Swan Lake Beta3 release.
+
+## Updating Ballerina
+
+If you are already using Ballerina, you can use the [Update Tool](/learn/tooling-guide/cli-tools/update-tool/) to directly update to Ballerina Swan Lake Beta4 as follows. 
+
+To do this, first, execute the command below to get the update tool updated to its latest version. 
+
+> `bal update`
+
+If you are using an **Update Tool version below 0.8.14**, execute the `ballerina update` command to update it. Next, execute the command below to update to Swan Lake Beta4.
+
+> `bal dist pull slbeta4`
+
+## Installing Ballerina
+
+If you have not installed Ballerina, then download the [installers](/downloads/#swanlake) to install.
```

Comment:
`installers` link seems to be broken. Can you please check?

Reply:
These are relative links and will work once rendered on the website.